glom/core.py

```diff
@@ -1440,62 +1469,21 @@ def __repr__(self):
         return '%s(%s)' % (cn, bbrepr(self.val))
 
 
-class _VType(object):
-    def __getattr__(self, attr):
-        return _SetVar(attr)
-
-
-V = _VType()
-
-
 class Vars(object):
     """
-    :class:`Vars` is where variables captured with `V` and SetVars() will be stored.
-    """
-    def __init__(self):
-        pass
-
-    def glomit(self, target, scope):
-        scope[V] = scope.__setitem__
-        return target
-
-    def __repr__(self):
-        return "Vars()"
-
-
-class _SetVar(object):
-    def __init__(self, name):
-        self.name = name
-
-    def glomit(self, target, scope):
-        scope[V](self.name, target)
-        return target
-
-    def __repr__(self):
-        cn = self.__class__.__name__
-        return 'V.%s' % self.name
-
-
-class SetVars(object):
-    """
-    Same behavior as :class:`Let`, but rather than setting variables
-    on the current scope, variables are set at the level of the next
-    enclosing :class:`Vars`
+    :class:`Vars` is a helper that stores variables
     """
     def __init__(self, **kw):
```

Comment:
how about a way to copy a dict (or another Vars()) in, kind of like the dict constructor?